jujugui/static/gui/src/app/store/env/legacy-api.js

```diff
@@ -3256,7 +3256,7 @@ YUI.add('juju-env-legacy-api', function(Y) {
 
   environments.endpointToName = endpointToName;
```

Comment:
I guess we can remove many of these helper functions already defined in api.js.
The legacy API in this case can safely reuse functions from api.js.